src/notify.js

```diff
@@ -59,6 +63,14 @@ function Notify(title, options = {}) {
     }
 }
 
+Notify.serviceWorkerSupported = function() {
```

Comment:
Well, the thing is that anyone using the Push API would have had to check for support way before notifications become an issue.

We could add a method, such as:

```javascript
Notify.isPushNotificationSupported = function() {
    return Notify.isServiceWorker() && self.registration.pushManager instanceof PushManager;
}
```

Not sure this library is where it belongs though, since it would need to be called way before notifications are of any real concern.